src/main/java/uk/gov/pay/api/utils/TrustStoreLoader.java

```diff
@@ -0,0 +1,64 @@
+package uk.gov.pay.api.utils;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.NoSuchFileException;
+import java.nio.file.Paths;
+import java.security.KeyStore;
+import java.security.KeyStoreException;
+import java.security.NoSuchAlgorithmException;
+import java.security.cert.Certificate;
+import java.security.cert.CertificateException;
+import java.security.cert.CertificateFactory;
+
+public class TrustStoreLoader {
```

Comment:
Isn't this a bit of an overkill for this task? 
  - We should try and avoid re-loading certs if they are already loaded. This might load it everytime the container starts up.   
 - My thoughts about this was Perhaps bash might be a better option in this case? And remove the certs after a successful load.